src/functionBased/pages/About.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+
+import React from 'react';
+import { Link, useRouteMatch, Route } from 'react-router-dom';
+import SinglePage from './SinglePage';
+
+const About = (props) => {
```

Comment:
Since you are using props in your components, adding validation will ensure you are using best practises. you can check this [article](https://reactjs.org/docs/typechecking-with-proptypes.html) to understand better how to add prop validations. [additional article](https://github.com/yannickcr/eslint-plugin-react/issues/2079) for solving forbidden prop validations